payments-core/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -70,6 +70,12 @@ sealed interface StripeIntent : StripeModel {
      */
     val linkFundingSources: List<String>
 
+    /**
+     * Country code of the user.
```

Comment:
Should this be country code of the merchant?

Reply:
I copied this from the payment intent/setup intent. So I _assume_ that was right. But now that you bring it up, that doesn't help my merchant country code issue that I'm trying to pass it into. Let me think about this.